### PR TITLE
Feat/admin invoices index

### PR DESCRIPTION
**Pull request by cballrun**
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please describe):
## Problem/feature
_What problem are you trying to solve?_
Creation of Admin Invoices Index
## Solution
_How did you solve the problem?_
Create Admin invoices index spec
Create Admin invoices controller
Create index method
Create route/view file/method for admin invoice show so links work
## Checklist
- [x] Code compiles correctly
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: spec/features/admin/invoices/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  #       within "inv_#{inv.id}" do
+  #         click_link(inv.id)
+  #         expect(current_path).to eq(admin_invoice_path(inv.id))
+  #       end
+  #     end
+  #   end
+  # end
+
+  it 'each id links to the admin invoice show page' do
+    click_link(@invoices[0].id)
```
Comment (candace-eckels) [inline spec/features/admin/invoices/index_spec.rb]: This you should change the click_link("#{@invoices[0].id}") to be in quotations, that will get rid of the  capybara warning

**Review by candace-eckels — COMMENTED** [review]
- Just need to put where the capybara warning is in quotations for that click link
- needs to have invoice id linked to the invoice show (User story 23)

**Comment by candace-eckels** [comment]
Also, needs to add model test for invoice_item model method "self.unshipped_invoice_items"

**Comment by cballrun** [comment]
I have added the changes we discussed in our standup

**Pull request merged**